### PR TITLE
[eclipse/xtext-xtend#397] Support for ternary expressions and try with resource

### PR DESCRIPTION
**Pull request by szarnekow**
[eclipse/xtext-xtend#397] Support for ternary expressions and try with resource

### CODE REVIEW

File: org.eclipse.xtext.xbase.ui/src-gen/org/eclipse/xtext/xbase/annotations/ui/AbstractXbaseWithAnnotationsUiModule.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2010-2017 itemis AG (http://www.itemis.eu) and others.
+ * Copyright (c) 2010, 2018 itemis AG (http://www.itemis.eu) and others.
```
Comment (cdietrich) [inline org.eclipse.xtext.xbase.ui/src-gen/org/eclipse/xtext/xbase/annotations/ui/AbstractXbaseWithAnnotationsUiModule.java]: wrong date (other places too)

File: releng/org.eclipse.xtext.target/org.eclipse.xtext.target-r201903.target [hunk 2]
```diff
@@ -62,7 +62,7 @@
 		<unit id="org.eclipse.xtext.xbase.testlanguages.source" version="0.0.0"/>
 		<unit id="org.eclipse.xtext.xbase.testlanguages.ide" version="0.0.0"/>
 		<unit id="org.eclipse.xtext.xbase.testlanguages.ide.source" version="0.0.0"/>
-		<repository location="https://ci.eclipse.org/xtext/job/xtext-extras/job/master/lastStableBuild/artifact/build/p2-repository/"/>
+		<repository location="https://ci.eclipse.org/xtext/job/xtext-extras/job/ep_issue397/lastStableBuild/artifact/build/p2-repository/"/>
```
Comment (cdietrich) [inline releng/org.eclipse.xtext.target/org.eclipse.xtext.target-r201903.target]: dont forget to adjust back

File: org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/contentassist/XbaseProposalProvider.java [hunk 3]
```diff
@@ -859,8 +865,8 @@ public IEObjectDescription apply(JvmFeature feature) {
 		if (Strings.isEmpty(pref)) {
 			return result;
 		}
-		String[] favourites= pref.split(";"); //$NON-NLS-1$
-		for(String fav : favourites) {
+		String[] favorites= pref.split(";"); //$NON-NLS-1$
```
Comment (cdietrich) [inline org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/contentassist/XbaseProposalProvider.java]: do we have a new mixed style for assignments in formatting?

File: org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/contentassist/XbaseProposalProvider.java [hunk 4]
```diff
@@ -904,7 +910,7 @@ public boolean apply(JvmFeature input) {
 	/**
 	 * @since 2.17
 	 */
-	protected void proposeFavoritStaticFeatures(EObject context,ContentAssistContext contentAssistContext,
+	protected void proposeFavoriteStaticFeatures(EObject context,ContentAssistContext contentAssistContext,
```
Comment (cdietrich) [inline org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/contentassist/XbaseProposalProvider.java]: [dietrich@dietrich-thinkpad-x230 xtext-umbrella]$ ./gitAll grep favourite | wc -l
53
[dietrich@dietrich-thinkpad-x230 xtext-umbrella]$ ./gitAll grep favorite | wc -l
49

seems like we cannot decide if we go for BE or AE

Comment (szarnekow) [reply org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/contentassist/XbaseProposalProvider.java]: :(

**Pull request merged**